src/psyclone/core/access_info.py

```diff
@@ -153,8 +153,14 @@ def __init__(self, signature):
 
     def __str__(self):
         '''Returns a string representation of this object with the format:
-        var_name:WRITE(2),WRITE(3),READ(5).'''
-        return "{0}:{1}".format(str(self._signature),
+        var_name:WRITE(2),WRITE(3),READ(5) where the numbers indicate
+        the 'location' of the corresponding access. The location is an
+        integer number that enumerates each stament in a program unit,
```

Comment:
"statement" and "if an access is earlier, ..." below.

Reply:
Done.